bin/pakiti-client

```diff
@@ -452,6 +471,8 @@ sub init () {
         "output"   => "|o=s",
         "rndsleep" => "|r=i",
         "site"     => "=s",
+        "svmonreport" => "|s",
+        "svmon"    => "=s",
```

Comment:
This specification is superfluous, it's added automatically later (see line 479), please remove it.